### DISCUSSION TITLE
Encountering rmr0201 Error When Using Remirror with Suspense

### DISCUSSION THREAD

**Discussion by kangyeop** [remirror/remirror#2259] [Q&A]
Hello everyone,

I've recently been integrating Remirror into my project and decided to use it alongside React's Suspense for data fetching. However, I've encountered an issue where the application throws an `rmr0201` error.

Here are the steps I followed:

1. Set up a basic Remirror editor.
2. Wrapped the editor component with React.Suspense.
3. Implemented data fetching logic within the suspense boundary.

Despite following the documentation, the rmr0201 error persists, and I'm not entirely sure what's causing it. Has anyone faced a similar issue or have any insights on how to resolve this error?

Any help or guidance would be greatly appreciated!

Thank you in advance.

**Comment by febkosq8** [comment]
Have been facing the same issue after switching my codebase into using React Suspense. If anyone has a workaround or a solution, it would be great !
